core/src/main/scala/kafka/log/LogCleanerManager.scala

```diff
@@ -39,6 +39,10 @@ private[log] case object LogCleaningInProgress extends LogCleaningState
 private[log] case object LogCleaningAborted extends LogCleaningState
 private[log] case class LogCleaningPaused(pausedCount: Int) extends LogCleaningState
 
+private[log] case class LogCleaningException(log: Log,
```

Comment:
Why not make this a normal class?

Reply:
Good call, that was probably a copy-paste from the classes above